Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -413,6 +413,8 @@ static void scheduleLogMemoryStatistics(LogMemoryStatisticsReason reason)
 void WebProcess::initializeWebProcess(WebProcessCreationParameters&& parameters)
 {    
     TraceScope traceScope(InitializeWebProcessStart, InitializeWebProcessEnd);
+    ProcessIdentity p { ProcessIdentity::CurrentProcess };
+    parentProcessConnection()->send(Messages::WebProcessProxy::InitializeIdentity(WTFMove(p)), 0);
```

Comment:
Maybe we should use sendWithAsyncReply for initializeWebProcess() and have it return the process identity?

Looks to me that `WebProcessProxy::InitializeIdentity` is only sent once.